tests/unit/amo/components/TestCategory.js

```diff
@@ -161,7 +164,8 @@ describe(__filename, () => {
     const fakeDispatch = sinon.stub(store, 'dispatch');
     render({}, { autoDispatchCategories: false });
 
-    sinon.assert.notCalled(fakeDispatch);
+    sinon.assert.callCount(fakeDispatch, 1);
```

Comment:
I'd like to avoid call count assertions unless absolutely necessary because they introduce a maintenance burden. For example, we may later add another unrelated dispatch to the component but that should not break this test.